packages/nodes-base/nodes/Hubspot/ContactDescription.ts

```diff
@@ -397,10 +397,7 @@ export const contactFields = [
 					},
 				},
 				default: '',
-				description: `Used to include specific company properties in the results.<br/>
-				By default, the results will only include company ID and will not include the values for any properties for your companys.<br/>
-				Including this parameter will include the data for the specified property in the results.<br/>
-				You can include this parameter multiple times to request multiple properties separed by ,.`,
+				description: `Used to include specific company properties in the results. By default, the results will only include company ID and will not include the values for any properties for your companys. Including this parameter will include the data for the specified property in the results. You can include this parameter multiple times to request multiple properties separated by ,.`,
```

Comment:
this file has all the same issues as in the other Hubspot comments. I wonder if `<code>,</code>` or `','` is better here